tests/model_geometry_test.js

```diff
@@ -8,331 +8,375 @@ const { expect } = require('chai'),
   semver = require('semver');
 
 const version_helper = require('../source/version_helper.js')
-const { makeTestSequelizeInstance } = require('./helper.js')
 
-function shouldSkip(test, isCRDBVersion21_2Plus) {
-  if (!isCRDBVersion21_2Plus) {
-    test.skip()
-  }
-}
+const CRDBVersion = version_helper.GetCockroachDBVersionFromEnvConfig()
+const IsCRDBVersion21_2Plus =  CRDBVersion ? semver.gte(CRDBVersion, "21.1.2") : false
```

Comment:
nit: if it's local, the first letter should be lower case (for both of these constants)